packages/client/src/components/GrantsTableNext.vue

```diff
@@ -277,6 +279,16 @@ export default {
         this.loading = false;
       }
     },
+    openSearchForEdit(searchId) {
```

Comment:
This can be passed a search ID to open in the panel, otherwise, it will open whatever search was last selected.